app/controllers/dashboard_controller.rb

```diff
@@ -2,5 +2,12 @@ class DashboardController < ApplicationController
   def index
     @language_user = LanguageUser.new
     @languages = current_user.languages
+
+    @entries = Entry.where(user_id: current_user.id)
+                    .order('id asc')
+                    .last(5)
+                    .map { |entry| 
+                      DashboardEntryPresenter.new(entry) 
+                    }
```

Comment:
If this collection needs any further functionality it would be good to extract this out into an aggregate object.